docker-compose.demo.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.8'
+services:
+  svc:
+    container_name: contrastservice
+    hostname: contrastservice
+    build:
+      context: .
+      dockerfile: Dockerfile.svc 
+    env_file:
+      - svc.env 
+    networks:
+      - extbridge
+  web:
+    build: 
+      context: .
+      dockerfile: Dockerfile.agent
+    ports:
+      - 8080:8080
+    env_file:
+      - svc.env
```

Comment:
I wouldn't expect the `svc.env` to be used outside of the `svc` service. Can we rename this to something more generic if it's shared by the service and the agent/app?

Reply:
I'll update this to a different name.